web/styles/settings.css

```diff
@@ -1743,6 +1743,9 @@ $option_title_width: 180px;
     }
 }
 
+.text_cursor {
```

Comment:
I think would be better to move this code in `app_components.css`.

Reply:
Moved it to `app_components.css`